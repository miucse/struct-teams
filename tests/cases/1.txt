5

Brutal Bytes
Manarat International University
9
Shamim Ahmed
Jakariya Abbas
Ahmad Hasan Mubashshir

Printf("hello");
Manarat International University
8
Mostakim Sakib
Irfan Khan
Saidur Rahman

team X
Manarat International University
5
Muhammad hasnat
Raihan Chowdhury
Sakib Al-Hashem

Stack Smashers
North South University
7
Raihan Chowdhury Abir
Shafin Raiyan
Nufayer Mahmud

Segfault Solvers
BRAC University
6
Ashfaqul Awal Himel
Zuhayer Mashrur Chowdhury
Suhail Mujtabir Fuad
