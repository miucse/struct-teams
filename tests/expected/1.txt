Team name  : Brutal Bytes
Institution: Manarat International University
Solved     : 9
   Member 1: Shamim Ahmed
   Member 2: Jakariya Abbas
   Member 3: Ahmad Hasan Mubashshir
